server/views/pages/personal-details.njk

```diff
@@ -238,6 +238,10 @@
             <p>
             <a href="/case/{{ crn }}/handoff/oasys">View sentence plan in OASys</a>
             </p>
+
+            <p>
+            <a href="{{ sentencePlanLink() }}" target="_blank">View sentence plan in UI</a>
```

Comment:
Should this say something other than "UI"?  Maybe something like "View in the sentence plan service"? I'm not sure....